python/requirements_linters.txt

```diff
@@ -2,6 +2,7 @@ flake8==3.9.1
 flake8-comprehensions
 flake8-quotes==2.0.0
 flake8-bugbear==21.9.2
-mypy==0.782
+mypy==0.982
+types-PyYAML==6.0.12.2
```

Comment:
This was needed since mypy is now doing a little more checking and seems to need this plugin now